install/linux/usr/bin/odemis-select-mic-start

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+: '
+This script is used when there are multiple modular configurations available for METEOR.
+It is recommended to combine it with the exec= command of the Odemis.desktop file.
+However it could be used stand-alone if it is required.
+The syntax to run this script -> odemis-select-mic-start [target_files].
+The script is typically used for parsing configuration (.yaml) files which contain a "Microscope" class.
+Using this class microscope names are extracted and placed in a list.
+This list is presented to the user in the form of a (zenity) selection dialog window.
+After selection of the microscope configuration the right (yaml) file will be used to start Odemis.
+'
+# extract the arguments which were used to start this script
+args=( "$@" )
+
+for arg in ${args[@]}; do
+    if test "$arg" == "--help"; then
+        # Display help on the argument usage
+        echo
+        echo "Syntax: odemis-select-mic-start [target_files]"
+        echo "arguments:"
+        echo "target_files      A selection of configuration files from a specific location."
+        echo "                  (e.g. /usr/share/odemis/sim/*meteor-main*.yaml)."
+        exit 0
+    fi
+done
+
+odemis-cli --check
+status=$?
+
+# check if the backend is already running or starting up
+if [ $status == 0 ] || [ $status == 3 ] ; then
+    # display an extra dialog for the user to make a choice cancel/restart backend/start gui only
+    reply=$(zenity --info --title="Odemis backend already running" --text "Restart with new backend or start GUI only" --ok-label="Cancel" --extra-button="Restart" --extra-button="GUI only" --width=300 --height=50)
+    if [ "$reply" == "Restart" ]; then
+        echo "Trying to stop the backend"
+        sudo odemis-stop
+    elif [ "$reply" == "GUI only" ]; then
+        echo "Trying to start the GUI"
+        odemis-gui
+        exit
+    else
+        exit
+    fi
+fi
+
+mic_list=()
+for fn in ${args[@]}; do
```

Comment:
I think you need to write `"${args[@]}"` to protect it from spaces. However, you can also simplify it and just write `for var in "$@"; do`  (no need to convert to an array).

Same thing for the --help search.